src/echoclient/EchoClient.php

```diff
@@ -163,12 +192,22 @@ public function getRecentEvents($class=null, $key=null, $value=null, $limit=25)
 
             if ($response->isSuccessful())
             {
-                $result = json_decode($response->getBody(true),true);
-                if (isset($result['events']))
-                {
-                    $this->getLogger()->debug('Success getting events from echo - '.$class);
-                    return $result['events'];
+                switch($format){
+                    case "csv":
+                        $result = $response->getBody(true);
+                        if($result){
+                            return $result;
+                        }
+                        break;
+                    default:
+                        $result = json_decode($response->getBody(true),true);
```

Comment:
I think that's okay.  If json_decode returns null then it will fail the next check and will not return the results from the method, instead throwing an error as per the previous behaviour.